docs/import/add_disaster.js

```diff
@@ -257,6 +262,28 @@ function createAssetPickers(states) {
   }
 }
 
+/**
+ * Deletes a disaster from firestore. Confirms first. Returns when deletion is
+ * complete (or instantly if deletion doesn't actually happen).
+ * @param {Window} globalWindow DOM Window (for test injection)
+ * @return {Promise<void>}
+ */
+function deleteDisaster(globalWindow = window) {
+  const disasterPicker = $('#disaster');
+  const disasterId = disasterPicker.val();
+  if (globalWindow.confirm(
+          'Delete ' + disasterId + '? This action cannot be undone')) {
+    disasters.delete(disasterId);
+    disasterPicker.val(disasterPicker.children().eq(0).val()).trigger('change');
```

Comment:
What is this for? And don't you do this somewhere else too? Extract to a function?

Reply:
This sets the picker to the most recent disaster once you delete the one you're looking at. I do it in the cancel PR as well (: I'll extract in whichever I submit second if that's alright with you.